python/pyiceberg/catalog/glue.py

```diff
@@ -0,0 +1,237 @@
+                    'Description': 'string',  # To be fixed
+                    'Owner': properties[OWNER] if properties and OWNER in properties
+                    else boto3.client("sts").get_caller_identity().get("Account"),
+                    'LastAccessTime': now,
+                    'LastAnalyzedTime': now,
+                    'StorageDescriptor': {
+                        'Columns': list(map(_convert_column, schema.fields)),
+                        'Location': location or 's3://',  # To be fixed
+                        'BucketColumns': [
+                            'string',
+                        ],
+                        'SortColumns': [{
+                            schema.find_column_name(field.source_id),
+                            1 if field.direction == SortDirection.ASC else 0
+                        } for field in sort_order.fields]
+                    },
+                    'PartitionKeys': [
+                        {
+                            'Name': schema.find_column_name(spec.source_id),
+                            'Type': str(schema.find_type(spec.source_id)),
+                            'Comment': str(spec.transform)
+                        }
+                        for spec in partition_spec.fields],
+                    'TableType': 'EXTERNAL_TABLE',
+                    'Parameters': properties,
+                }
+            )
+
+            load_table_response = self.client.get_table(DatabaseName=database_name, Name=table_name)
+            print(load_table_response)
+            return load_table_response['Table']  # TODO: convert glue to iceberg
+        except self.client.exceptions.AlreadyExistsException:
+            # TODO log already exists info
+            return None
+
+    # tested
+    def load_table(self, identifier: str | Identifier) -> Table:
+        database_name, table_name = self.identifier_to_tuple(identifier)
+        try:
+            load_table_response = self.client.get_table(DatabaseName=database_name, Name=table_name)
+            self._check_response(load_table_response)
+            return load_table_response['Table']
+        except self.client.exceptions.EntityNotFoundException:
+            # TODO: log not found message
+            return None
+
+    def drop_table(self, identifier: str | Identifier) -> None:
+        database_name, table_name = self.identifier_to_tuple(identifier)
+        self.client.delete_table(
+            DatabaseName=database_name,
+            Name=table_name
+        )
+        return None
+
+    def purge_table(self, identifier: str | Identifier) -> None:
+        self.drop_table(identifier)
+
+    def rename_table(self, from_identifier: str | Identifier, to_identifier: str | Identifier) -> Table:
+        raise NotImplementedError("AWS Glue does not support renaming of tables")
+
+    # tested but cannot see on glueCatalog
+    def create_namespace(self, namespace: str | Identifier, properties: Properties = EMPTY_DICT) -> None:
```

Comment:
Created database cannot be viewed on GlueCatalog UI, but they do exists since if trying to create a database with the same name on console, we see an AlreadyExistsError